addons/website/static/src/scss/website.scss

```diff
@@ -443,6 +443,7 @@ table.table_desc tr td {
         cursor: pointer;
         width: 8%;
         opacity: 1;
+        margin: 0 1px;
```

Comment:
- "When applied this commit will... no bottom scroll bar when use carousel" -> title is not good
- "displays the horizontal scroll bar because the size of the carousel arrows are in percentage" -> not true, set it in pixel, it does not fix the bug at all
- Tested on Firefox, no bug so I wondered why Chrome really needs that margin and I debugged: the root cause is not the arrow being offscreen, this is the `sr-only` item inside the arrow which is positioned absolutely, outside of the arrow => the fix should be done on that item. I guess it could even be fixed in bootstrap_review.scss with something like this (not tested):
```scss
.carousel-control-prev,
.carousel-control-next {
    .sr-only {
        left: 50%;
    }
}
```
That would fix the issue generically and avoid a lot of pain forward-porting this.
- In any case, needs a comment in the scss

Reply:
@qsm-odoo I tested, your fix works, I changed the commit and the message. But, for the change, the bootstrap_review.scss file does not exist in 12.0

Reply:
https://github.com/odoo/odoo/blob/12.0/addons/web/static/src/scss/bootstrap_review.scss

Reply:
:+1: